Packs/ExpanseV2/Integrations/ExpanseV2/ExpanseV2.py

```diff
@@ -1988,6 +2001,11 @@ def get_domain_command(client: Client, args: Dict[str, Any]) -> List[CommandResu
 
 
 def get_cloud_resource_command(client: Client, args: Dict[str, Any]) -> List[CommandResults]:
+    asset_id: Optional[str] = args.pop('id', None)
+
+    if asset_id is not None:
+        output = client.get_cloud_resource(asset_id=asset_id)
+        return format_cloud_resource_data([output])
```

Comment:
Does the `output` result be a dictionary in all cases? If not, please change as suggested.
```suggestion
        output = output if isinstance(output, list) else [output]
        return format_cloud_resource_data(output)
```